package.json

```diff
@@ -56,6 +57,8 @@
     "@types/node": "20.4.10",
     "@typescript-eslint/eslint-plugin": "5.6.0",
     "@typescript-eslint/parser": "5.6.0",
+    "bitcoin-json-rpc": "1.3.3",
+    "bw-electrum-client": "BlueWallet/rn-electrum-client#47acb51149e97fab249c3f8a314f708dbee4fb6e",
```

Comment:
Why can't we use our client?

Reply:
For now I just copy-pasted the solution I'm using in the bitkit test, we can improve it later, np